examples/react/main.js

```diff
@@ -8,7 +8,7 @@ class Widget extends Component {
 }
 
 Widget.propTypes = {
-  name: string.isRequired
+  name: string.isRequired,
```

Comment:
If we don't want to have trailing commas, we can configure the Prettier rule with `trailingComma: 'none'`.  I thought I'd start out by enforcing trailing commas though.  I've come to appreciate how much easier it is to edit multi-line object literals and arrays when they are used.

Reply:
Do our transpile steps properly remove trailing commas? If not, enforcing leading commas would also be an option. But I'm fine with either.